cmd/influx/task.go

```diff
@@ -268,8 +268,16 @@ func (b *cmdTaskBuilder) taskRetryFailedF(*cobra.Command, []string) error {
 		} else {
 			fmt.Printf("Would retry for %s run for Task %s.\n", run.ID, run.TaskID)
 		}
-
 	}
+	if !b.taskRerunFailedFlags.executeRuns {
+		uniqueIDs := make(map[influxdb.ID]struct{})
+		for _, r := range failedRuns {
+			uniqueIDs[r.TaskID] = struct{}{}
+		}
+		fmt.Printf("Dry run complete. Found %d tasks with a total of %d runs to be retried\n"+
+			"Rerun with '--run true' to execute", len(uniqueIDs), len(failedRuns))
```

Comment:
Let me know if this is too much work to find the number of tasks that had failed runs. I figured more info for the user the better

Reply:
This is great! 

In Go `len` doesn't require any significant work; the length is tracked within the slice structure not calculated on demand so you're good there.